{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/.pre-commit-config.yml

```diff
@@ -0,0 +1,8 @@
+exclude: "\\.asdf$"
+repos:
+  # This should be before any formatting hooks
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: "v0.1.15"
+    hooks:
+      - id: ruff
+        args: ["--fix"]
```

Comment:
Do we want to discuss ruff formatting here?

Reply:
Sure, what do you have in mind specifically?